mobile/lib/entities/user.entity.dart

```diff
@@ -36,7 +36,7 @@ class User {
         isPartnerSharedWith = false,
         profileImagePath = dto.profileImagePath,
         isAdmin = dto.isAdmin,
-        memoryEnabled = dto.memoriesEnabled ?? false,
+        memoryEnabled = false,
```

Comment:
@alextran1502 can you help me fix this? `memoryEnabled` now comes from `userApi.getMyPreferences()`.